src/guide/1/essentials/security/index.md

```diff
@@ -54,20 +54,20 @@ All `roles` and `profiles` can be edited in the [Kuzzle Admin Console]({{ site_b
 
 A `role` can be defined using a hierarchical JSON object where permissions are outlined by `controller` and `action`.
 
-The `role` definition is represented as a Javascript object where each key at the root of the object identifies a `controller` by name:
+The `role` definition is represented as a JSON object where each key at the root of the object identifies a `controller` by name.
 
 ```js
-const myRoleDefinition = {
-  controllers: {
-    < controller|* >: {
-      actions: {
-        < action|* >: < action permission|* >,
-        < action|* >: < action permission|* >,
-        ...
+{
+  "controllers": {
+    "<controller name|*>": {
+      "actions": {
+        "<action name|*>": true, // accepted values: [true|false|object]
+        "<action name|*>": false,
+        // ...
       }
     }
   }
-};
+}
 ```
```

Comment:
My comment is about the unchanged text below:
```
The `action permission` value can be set to either:
- a boolean. If `true`, the `role` allows the given action.
- an object describing a dynamic right definition. For more information check out the [advanced roles documentation]({{ site_base_path }}guide/1/kuzzle-depth/roles-definitions)).
```

As the closures are deprecated, do we really want to reference it here ? Or at least add a `deprecated` tag to the line `an object describing a dynamic right definition`.

Reply:
I added a `deprecated` tag, because a documentation must be exhaustive, even if it means giving out of date details.